webrtc-sys/libwebrtc/boringssl_prefix_symbols.txt

```diff
@@ -3441,196 +2693,45 @@ i2s_ASN1_OCTET_STRING LK_i2s_ASN1_OCTET_STRING
-rsaz_1024_norm2red_avx2 LK_rsaz_1024_norm2red_avx2
-rsaz_1024_red2norm_avx2 LK_rsaz_1024_red2norm_avx2
-rsaz_1024_scatter5_avx2 LK_rsaz_1024_scatter5_avx2
-rsaz_1024_sqr_avx2 LK_rsaz_1024_sqr_avx2
 s2i_ASN1_INTEGER LK_s2i_ASN1_INTEGER
 s2i_ASN1_OCTET_STRING LK_s2i_ASN1_OCTET_STRING
-sha1_block_data_order_avx LK_sha1_block_data_order_avx
-sha1_block_data_order_avx2 LK_sha1_block_data_order_avx2
-sha1_block_data_order_hw LK_sha1_block_data_order_hw
-sha1_block_data_order_nohw LK_sha1_block_data_order_nohw
-sha1_block_data_order_ssse3 LK_sha1_block_data_order_ssse3
-sha256_block_data_order_avx LK_sha256_block_data_order_avx
-sha256_block_data_order_hw LK_sha256_block_data_order_hw
-sha256_block_data_order_nohw LK_sha256_block_data_order_nohw
-sha256_block_data_order_ssse3 LK_sha256_block_data_order_ssse3
-sha512_block_data_order_avx LK_sha512_block_data_order_avx
-sha512_block_data_order_nohw LK_sha512_block_data_order_nohw
+size_t LK_size_t
+sk_deep_copy LK_sk_deep_copy
+sk_delete LK_sk_delete
+sk_delete_if LK_sk_delete_if
+sk_delete_ptr LK_sk_delete_ptr
+sk_dup LK_sk_dup
+sk_find LK_sk_find
 sk_free LK_sk_free
+sk_insert LK_sk_insert
+sk_is_sorted LK_sk_is_sorted
+sk_new LK_sk_new
 sk_new_null LK_sk_new_null
 sk_num LK_sk_num
 sk_pop LK_sk_pop
 sk_pop_free LK_sk_pop_free
 sk_pop_free_ex LK_sk_pop_free_ex
 sk_push LK_sk_push
+sk_set LK_sk_set
+sk_set_cmp_func LK_sk_set_cmp_func
+sk_shift LK_sk_shift
+sk_sort LK_sk_sort
 sk_value LK_sk_value
-spx_base_b LK_spx_base_b
-spx_copy_keypair_addr LK_spx_copy_keypair_addr
-spx_fors_pk_from_sig LK_spx_fors_pk_from_sig
-spx_fors_sign LK_spx_fors_sign
-spx_fors_sk_gen LK_spx_fors_sk_gen
-spx_fors_treehash LK_spx_fors_treehash
-spx_get_tree_index LK_spx_get_tree_index
-spx_ht_sign LK_spx_ht_sign
-spx_ht_verify LK_spx_ht_verify
-spx_set_chain_addr LK_spx_set_chain_addr
-spx_set_hash_addr LK_spx_set_hash_addr
-spx_set_keypair_addr LK_spx_set_keypair_addr
-spx_set_layer_addr LK_spx_set_layer_addr
-spx_set_tree_addr LK_spx_set_tree_addr
-spx_set_tree_height LK_spx_set_tree_height
-spx_set_tree_index LK_spx_set_tree_index
-spx_set_type LK_spx_set_type
-spx_thash_f LK_spx_thash_f
-spx_thash_h LK_spx_thash_h
-spx_thash_hmsg LK_spx_thash_hmsg
-spx_thash_prf LK_spx_thash_prf
-spx_thash_prfmsg LK_spx_thash_prfmsg
-spx_thash_tk LK_spx_thash_tk
-spx_thash_tl LK_spx_thash_tl
-spx_to_uint64 LK_spx_to_uint64
-spx_treehash LK_spx_treehash
-spx_uint64_to_len_bytes LK_spx_uint64_to_len_bytes
-spx_wots_pk_from_sig LK_spx_wots_pk_from_sig
-spx_wots_pk_gen LK_spx_wots_pk_gen
-spx_wots_sign LK_spx_wots_sign
-spx_xmss_pk_from_sig LK_spx_xmss_pk_from_sig
-spx_xmss_sign LK_spx_xmss_sign
+sk_zero LK_sk_zero
+ssl_cert_check_key_usage LK_ssl_cert_check_key_usage
+ssl_client_hello_init LK_ssl_client_hello_init
+ssl_decode_client_hello_inner LK_ssl_decode_client_hello_inner
+ssl_is_valid_ech_public_name LK_ssl_is_valid_ech_public_name
+ssl_session_serialize LK_ssl_session_serialize
+uint16_t LK_uint16_t
```

Comment:
this doesn't seems right? how are you extracting the types?

Reply:
oops yeah, this one has a newline in the definition so it got the type. i'll have a manual comb through for those

Reply:
i'll add it commented out to the top of the symbol rename list